packages/store-api/package.json

```diff
@@ -13,20 +13,26 @@
     "node": ">=10"
   },
   "scripts": {
-    "develop": "tsdx watch",
-    "build": "tsdx build",
+    "develop":  "concurrently \"yarn generate -w\" \"tsdx watch\"",
+    "build": "graphql-codegen --config codegen.yml && tsdx build",
     "test": "tsdx test",
-    "lint": "tsdx lint"
+    "lint": "tsdx lint",
+    "generate": "graphql-codegen --config codegen.yml"
   },
   "dependencies": {
+    "@graphql-tools/schema": "^8.2.0",
+    "graphql": "^15.5.3",
+    "isomorphic-unfetch": "^3.1.0",
```

Comment:
Why were these dependencies added?

Reply:
It was already importing these on the code but it wasn't really defined in store-api's deps.

Reply:
Can we prevent that from hapenning? Like, scoping dependencies per package?